packages/params/package.json

```diff
@@ -52,7 +52,8 @@
     "check-types": "tsc",
     "lint": "eslint --color --ext .ts src/ test/",
     "lint:fix": "yarn run lint --fix",
-    "test": "yarn run check-types",
+    "pretest": "yarn run check-types",
```

Comment:
Why do we have to pre hook at all? I have to admit I never executed `yarn test` as I usually just run specific tests locally (mostly `yarn test:unit`).

From the contribution docs, there is already a reference to run type checks
- :test_tube: Run `yarn check-types` to check TypeScript types.

Reply:
Good point. I guess the benefit is being transparently called.
Now as you said, it doesn't appear that `yarn test` is used much.

Maybe both could be remove altogether.

Reply:
Feel like we can keep test command in case someone just wants to execute all tests. Not convinced by the pretest hook though

Reply:
Removed `pretest` scripts.

Reply:
In general devs used to just run `test` script and some IDE integrations also use it, so having a `pretest` with the type-check is not harmful. https://github.com/ChainSafe/lodestar/pull/6339#discussion_r1464717234

Reply:
Not strong feelings for me. One extra point is that adding `check-types` as `pretest` slightly increase execution time.